InventoryBuilder.py

```diff
@@ -129,7 +129,11 @@ def get_vrops(self):
         vrops_list = list()
         for target in netbox_json:
             if target['labels']['job'] == "vrops":
-                vrops = target['labels']['server_name']
+                vrops = types.SimpleNamespace()
+                vrops.server_name = target['labels']['server_name']
+                vrops.username    = target['labels'].get('username', None)
+                vrops.password    = target['labels'].get('password', None)
+                vrops.auth_source = target['labels'].get('auth_source', None)
                 vrops_list.append(vrops)
```

Comment:
Leveraging `target['labels']` to introduce username, pw and auth_source is not ideal. Since we are discussing #80 already I would propose the following:
Configure self.auth_source in InventoryBuilder globally and add the possibility to set it via ENV/CLI like we do everywhere, defaulting to `Local`.
That should enable you to use your auth source. Let's have the configfile / credential solution discussed first.

Reply:
Yes. Once we have a config-file, we should move it there

Reply:
Just remove it and use the global var, if that wouldn't work for you, we can leave it in here for time being. Please add a `TODO: ..` here.

Reply:
Please decide for either one option.